security/network_defense.adoc

```diff
@@ -0,0 +1,78 @@
+[[security-network]]
+= Network Defense
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:sourcedir: ../
+
+toc::[]
+
+[[security-network-isolating-applications]]
+== Isolating Applications from Each Other
+
+Deploying modern microservices-based applications in containers often means
+deploying multiple containers distributed across multiple nodes. You need a way
+to isolate applications from other applications within a cluster. You need to
+consider network defense.
+
+A typical public cloud container service, such as Google Container Engine (GKE),
+Azure Container Services, and AWS Container Service, are single tenant services.
+They let you run your containers on the virtual machine (VM) cluster that you
+spin up. {product-title} allows you to take a single Kubernetes cluster and
+segment the traffic to make multi-tenant clusters the isolate different users,
+teams, applications and environments within a single cluster.
+
+[[security-network-namespaces]]
+== Network Namespaces
+
+The first line in network defense comes from network namespaces. With network
+namespaces, each collection of containers (each pod) gets its own entire IP and
+port range to bind to, thereby isolating pod networks from each other on the
+node. Pods from different projects cannot send packets to or receive packets
+from pods and services of a different project. You can use these features to
```

Comment:
We need to call out when using our multi-tennant plugin (its now the default, but was not with some versions of OpenShift). An then highlight that further or better granularity to this is offered by other plugins.